packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -496,18 +493,18 @@ export function messageChannelMiddleware(
       const breakpoints = ReactPage.getBreakpoints(state)
       messageChannel.port1.postMessage(setBreakpoints(breakpoints))
 
-      const routerLocale = Router.locale
-      if (routerLocale != null) {
-        messageChannel.port1.postMessage(setLocale(new Intl.Locale(routerLocale)))
-      }
+      // const routerLocale = Router.locale
+      // if (routerLocale != null) {
+      //   messageChannel.port1.postMessage(setLocale(new Intl.Locale(routerLocale)))
+      // }
 
-      Router.events.on('routeChangeStart', () => {
-        messageChannel.port1.postMessage(changePathnameStart())
-      })
+      // Router.events.on('routeChangeStart', () => {
+      //   messageChannel.port1.postMessage(changePathnameStart())
+      // })
 
-      Router.events.on('routeChangeComplete', () => {
-        messageChannel.port1.postMessage(changePathnameComplete())
-      })
```

Comment:
We might be able to remove this. This was added when we tried to add client-side navigation. But we're not even enabling the feature flag for this.

Reply:
Hmm, so that feature flag is set to off for all users? I think we might still need to figure out how we'll handle stuff like this but if we punt this for later then perhaps we should 🤔

Reply:
Yes. They're off for all users right now
<img width="1107" alt="image" src="https://github.com/makeswift/makeswift/assets/13066728/116b76d7-55af-4437-886d-fbfc412f1180">

Reply:
Update: I just tried clicking a link on a page using interaction mode, and it navigates me to the correct page in the builder. So client-side-navigation is working even when we disable the flag.

Reply:
So you're saying the flag isn't being used?

Reply:
Yes in a way. There are still some references to the flag, but the core functionality is still working without the flag.

Reply:
Ok, that's good to know. @chanceaclark and I were wondering whether we could just delete this and move on but it sounds like we might not want to do that.